diff --git a/dom/security/nsContentSecurityManager.cpp b/dom/security/nsContentSecurityManager.cpp
index 524d0cf7dd..f05c7ccbac 100644
--- a/dom/security/nsContentSecurityManager.cpp
+++ b/dom/security/nsContentSecurityManager.cpp
@@ -1045,6 +1045,10 @@ nsresult nsContentSecurityManager::CheckAllowLoadInSystemPrivilegedContext(
   // pref check & data sanitization happens in the called function
   MeasureUnexpectedPrivilegedLoads(loadInfo, innerURI, remoteType);
 
+  // DEBUG BUILD ONLY: Allow all remote loads from SystemPrincipal
+  // This patch is only applied in debug builds for development purposes
+  return NS_OK;
+
   // Relaxing restrictions for our test suites:
   // (1) AreNonLocalConnectionsDisabled() disables network, so
   // http://mochitest is actually local and allowed. (2) The marionette test
