diff --git a/browser/installer/windows/nsis/shared.nsh b/browser/installer/windows/nsis/shared.nsh
index fc11711dd6..74878900b8 100755
--- a/browser/installer/windows/nsis/shared.nsh
+++ b/browser/installer/windows/nsis/shared.nsh
@@ -880,6 +880,8 @@ FunctionEnd
     ${WriteRegDWORD2} $RegHive "$0" "ESR" 1 0
   ${EndIf}
 
+  ${WriteRegStr2} $RegHive "$0" "Publisher" "Ablaze & Floorp Community" 0
+
   StrCpy $0 "Software\Mozilla\${BrandFullNameInternal}$3"
   ${WriteRegStr2} $RegHive "$0" "" "${GREVersion}" 0
   ${WriteRegStr2} $RegHive "$0" "CurrentVersion" "${AppVersion}$3 (${ARCH} ${AB_CD})" 0
